languages/python/config.json

```
@@ -8,7 +8,14 @@
     "indent_size": 4
   },
   "exercises": {
-    "concept": [],
+    "concept": [
+      {
+        "slug": "strings",
+        "uuid": "c9ca5c4ec8f04c36812c721158e0d208",
+        "concepts": ["strings-basic"],
+        "prerequisites": ["functions"]
```

As functions are such a key component of Python, you'll probably want multiple exercises dealing with functions. As such, maybe use:

```suggestion
        "prerequisites": ["functions-basic"]
```